src/test/java/racing/core/CarTest.java

```diff
@@ -0,0 +1,54 @@
+package racing.core;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarTest {
+
+    private Car car = new Car();
+
+    @Test
+    @DisplayName("이동 성공 테스트")
+    void moveSuccess() {
+        Move movement = new AlwaysMove();
+        assertThat(car.move(movement)).isEqualTo(new Snapshot(1));
```

Comment:
람다하면 아래처럼 사용할 수도 있습니다:)
```suggestion
        assertThat(car.move(() -> true)).isEqualTo(new Snapshot(1));
```

Reply:
`AlwaysMove`를 클래스로 정의할 필요가 없었겠네요!